kuma/contributions/jinja2/contributions/contribute.html

```diff
@@ -0,0 +1,228 @@
+                    {% endtrans %}
+                </p>
+                <span class="small">{{_('Did this answer your question?')}}</span>
+                <div class="vote-button-group">
+                    <button class="thumbs-up" data-faq="6">{% include 'includes/icons/emojis/thumbs-up.svg' %}</button>
+                    <button class="thumbs-down" data-faq="6">{% include 'includes/icons/emojis/thumbs-down.svg' %}</button>
+                </div>
+            </li>
+            <li class="faq">
+                <h3>{{_('How does Mozilla make money?')}}</h3>
+                <span class="faq-number">7</span>
+                <p>
+                    {% trans url="https://www.mozilla.org/en-US/foundation/annualreport/2016/" %}
+                        Revenue sharing with search providers on Firefox’s
+                        search bar and start page (i.e. Google and others).
+                        Mozilla’s revenue mix also includes e-commerce
+                        affiliate revenue from online retailers (i.e. Amazon
+                        and others). As a nonprofit, we report these revenues
+                        publicly every trailing year, as in our most recent
+                        <a href="{{ url }}">2016 Mozilla Annual Report</a>.
+                    {% endtrans %}
+                </p>
+                <span class="small">{{_('Did this answer your question?')}}</span>
+                <div class="vote-button-group">
+                    <button class="thumbs-up" data-faq="7">{% include 'includes/icons/emojis/thumbs-up.svg' %}</button>
+                    <button class="thumbs-down" data-faq="7">{% include 'includes/icons/emojis/thumbs-down.svg' %}</button>
+                </div>
+            </li>
+            <li class="faq">
+                <h3>{{_('Why is money so important and can we one day untether ourselves from our financial overlords?')}}</h3>
+                <span class="faq-number">8</span>
+                <p>
+                    {% trans %}
+                        You ask a lot of good questions, and we like your
+                        attitude.  The tragedy of the commons requires an
+                        uncommon vigilance from people like us, online and off.
+                    {% endtrans %}
+                </p>
+                <span class="small">{{_('Did this answer your question?')}}</span>
+                <div class="vote-button-group">
+                    <button class="thumbs-up" data-faq="8">{% include 'includes/icons/emojis/thumbs-up.svg' %}</button>
+                    <button class="thumbs-down" data-faq="8">{% include 'includes/icons/emojis/thumbs-down.svg' %}</button>
+                </div>
+            </li>
+        </ul>
+    </div>
+    <div class="section">
+        <div class="index">
+            <h2>{{_('Feedback')}}</h2>
+        </div>
+        <div class="faqs clear feedback">
+            <form id="faq-feedback" class="faq-feedback">
```

Comment:
Missing `method` and `action` attributes?

Reply:
There is no method or action as it is posting to analytics, form is controlled by JS.